content/kotlin/concepts/datetime/terms/toDateTimePeriod/toDateTimePeriod.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: '.toDateTimePeriod()'
+Description: 'Interprets an ISO-8601 formatted duration string, converting it into a DateTimePeriod object. In cases where the string's time components are either missing or amount to zero, the function yields a DatePeriod instead.'
+Subjects:
+  - 'Computer Science'
+  - 'Software Engineering'
+  - 'Other comparable degrees'
+Tags:
+  - 'Functions'
+  - 'Data Structures'
+  - 'DatePeriod'
```

Comment:
This is not present [here](https://github.com/Codecademy/docs/blob/main/documentation/tags.md). Either add `DatePeriod` in `tags.md` file or use the tag `Date` instead.